### PR TITLE
Remove redundant placeholders in forms docs

### LINKED ISSUE #28260: Useless Placeholder Text on Forms Page
I don't see the need (or encourage anyone) to use placeholder text that just says the same thing as it's label.  Usability studies have shown that using placeholder text in this manner makes some users feel like the field has already been filled out and thus, may glance over it [[source]](https://www.amazon.com/Designing-UX-Forms-Create-Aspects/dp/0994347057/ref=sr_1_1?keywords=ux+forms&qid=1550162760&s=gateway&sr=8-1).  Here's an example from the [Readonly plain text](https://getbootstrap.com/docs/4.3/components/forms/#readonly-plain-text) section:

example

### PR DESCRIPTION
**Pull request by tojimjiang**
Fixes #28260

### CODE REVIEW

File: site/docs/4.3/components/forms.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Here's a quick example to demonstrate Bootstrap's form styles. Keep reading for
   </div>
   <div class="form-group">
     <label for="exampleInputPassword1">Password</label>
-    <input type="password" class="form-control" id="exampleInputPassword1" placeholder="Password">
+    <input type="password" class="form-control" id="exampleInputPassword1" placeholder="Enter password">
```
Comment (patrickhlauke) [inline site/docs/4.3/components/forms.md]: i'd remove this placeholder altogether, as well as the `placeholder="Enter email"` in the previous input

File: site/docs/4.3/components/forms.md [hunk 2]
```diff
@@ -799,21 +799,21 @@ Custom feedback styles apply custom colors, borders, focus styles, and backgroun
...
-      <input type="text" class="form-control" id="validationCustom03" placeholder="City" required>
+      <input type="text" class="form-control" id="validationCustom03" required>
       <div class="invalid-feedback">
         Please provide a valid city.
       </div>
     </div>
     <div class="col-md-3 mb-3">
       <label for="validationCustom04">State</label>
-      <input type="text" class="form-control" id="validationCustom04" placeholder="State" required>
+      <input type="text" class="form-control" id="validationCustom04" placeholder="Choose..." required>
```
Comment (patrickhlauke) [inline site/docs/4.3/components/forms.md]: "choose..." as placeholder for a text input doesn't make sense. for consistency with the previous form earlier in the page, suggest replacing this with an actual `<select>` instead

File: site/docs/4.3/components/forms.md [hunk 3]
```diff
@@ -867,34 +867,34 @@ While these feedback styles cannot be styled with CSS, you can still customize t
...
   <div class="form-row">
     <div class="col-md-6 mb-3">
       <label for="validationDefault03">City</label>
-      <input type="text" class="form-control" id="validationDefault03" placeholder="City" required>
+      <input type="text" class="form-control" id="validationDefault03" required>
     </div>
     <div class="col-md-3 mb-3">
       <label for="validationDefault04">State</label>
-      <input type="text" class="form-control" id="validationDefault04" placeholder="State" required>
+      <input type="text" class="form-control" id="validationDefault04" placeholder="Choose..." required>
```
Comment (patrickhlauke) [inline site/docs/4.3/components/forms.md]: "choose..." as placeholder for a text input doesn't make sense. for consistency with the previous form earlier in the page, suggest replacing this with an actual `<select>` instead

File: site/docs/4.3/components/forms.md [hunk 4]
```diff
@@ -947,21 +947,21 @@ We recommend using client-side validation, but in case you require server-side v
...
-      <input type="text" class="form-control is-invalid" id="validationServer03" placeholder="City" required>
+      <input type="text" class="form-control is-invalid" id="validationServer03" required>
       <div class="invalid-feedback">
         Please provide a valid city.
       </div>
     </div>
     <div class="col-md-3 mb-3">
       <label for="validationServer04">State</label>
-      <input type="text" class="form-control is-invalid" id="validationServer04" placeholder="State" required>
+      <input type="text" class="form-control is-invalid" id="validationServer04" placeholder="Choose..." required>
```
Comment (patrickhlauke) [inline site/docs/4.3/components/forms.md]: "choose..." as placeholder for a text input doesn't make sense. for consistency with the previous form earlier in the page, suggest replacing this with an actual `<select>` instead

File: site/docs/4.3/components/forms.md [hunk 5]
```diff
@@ -1075,21 +1075,21 @@ If your form layout allows it, you can swap the `.{valid|invalid}-feedback` clas
...
-      <input type="text" class="form-control" id="validationTooltip03" placeholder="City" required>
+      <input type="text" class="form-control" id="validationTooltip03" required>
       <div class="invalid-tooltip">
         Please provide a valid city.
       </div>
     </div>
     <div class="col-md-3 mb-3">
       <label for="validationTooltip04">State</label>
-      <input type="text" class="form-control" id="validationTooltip04" placeholder="State" required>
+      <input type="text" class="form-control" id="validationTooltip04" placeholder="Choose..." required>
```
Comment (patrickhlauke) [inline site/docs/4.3/components/forms.md]: "choose..." as placeholder for a text input doesn't make sense. for consistency with the previous form earlier in the page, suggest replacing this with an actual `<select>` instead

**Review by patrickhlauke — CHANGES_REQUESTED** [review]

File: site/docs/4.3/components/forms.md (hunk 5 above)
Comment (tojimjiang) [reply site/docs/4.3/components/forms.md]: These 4 have been changed to `<select>`.

File: site/docs/4.3/components/forms.md [hunk 6]
```diff
@@ -807,7 +807,7 @@ Custom feedback styles apply custom colors, borders, focus styles, and backgroun
     <div class="col-md-3 mb-3">
       <label for="validationCustom04">State</label>
       <select id="inputState" class="form-control" id="validationCustom04" required>
```
Comment (patrickhlauke) [inline site/docs/4.3/components/forms.md]: per the description just before the example/code block, best to also change the class to `class="custom-select"`

Comment (patrickhlauke) [reply site/docs/4.3/components/forms.md]: remove the redundant `id="inputState"`

File: site/docs/4.3/components/forms.md [hunk 7]
```diff
@@ -961,7 +961,7 @@ We recommend using client-side validation, but in case you require server-side v
     <div class="col-md-3 mb-3">
       <label for="validationServer04">State</label>
       <select class="form-control is-invalid" id="validationServer04" required>
```
Comment (patrickhlauke) [inline site/docs/4.3/components/forms.md]: again, change `class="form-control"` to `class="custom-select"`

File: site/docs/4.3/components/forms.md [hunk 8]
```diff
@@ -1092,7 +1092,7 @@ If your form layout allows it, you can swap the `.{valid|invalid}-feedback` clas
     <div class="col-md-3 mb-3">
       <label for="validationTooltip04">State</label>
       <select class="form-control" id="validationTooltip04" required>
```
Comment (patrickhlauke) [inline site/docs/4.3/components/forms.md]: and again `class="custom-select"`

**Review by patrickhlauke — CHANGES_REQUESTED** [review]
sorry, last few minor changes. then it looks good to go :+1:

File: site/docs/4.3/components/forms.md [hunk 9]
```diff
@@ -799,21 +799,24 @@ Custom feedback styles apply custom colors, borders, focus styles, and backgroun
...
+      <input type="text" class="form-control" id="validationCustom03" required>
       <div class="invalid-feedback">
         Please provide a valid city.
       </div>
     </div>
     <div class="col-md-3 mb-3">
       <label for="validationCustom04">State</label>
-      <input type="text" class="form-control" id="validationCustom04" placeholder="State" required>
+      <select id="inputState" class="form-control" id="validationCustom04" required>
+        <option selected disabled value = ''>Choose...</option>
```
Comment (mdo) [inline site/docs/4.3/components/forms.md]:
```suggestion
        <option selected disabled value='">Choose...</option>
```

File: site/docs/4.3/components/forms.md [hunk 10]
```diff
@@ -867,34 +870,37 @@ While these feedback styles cannot be styled with CSS, you can still customize t
...
     <div class="col-md-6 mb-3">
       <label for="validationDefault03">City</label>
-      <input type="text" class="form-control" id="validationDefault03" placeholder="City" required>
+      <input type="text" class="form-control" id="validationDefault03" required>
     </div>
     <div class="col-md-3 mb-3">
       <label for="validationDefault04">State</label>
-      <input type="text" class="form-control" id="validationDefault04" placeholder="State" required>
+      <select class="form-control" id="validationDefault04" required>
+        <option selected disabled value = ''>Choose...</option>
```
Comment (mdo) [inline site/docs/4.3/components/forms.md]:
```suggestion
        <option selected disabled value="">Choose...</option>
```

File: site/docs/4.3/components/forms.md [hunk 11]
```diff
@@ -947,21 +953,24 @@ We recommend using client-side validation, but in case you require server-side v
...
+      <input type="text" class="form-control is-invalid" id="validationServer03" required>
       <div class="invalid-feedback">
         Please provide a valid city.
       </div>
     </div>
     <div class="col-md-3 mb-3">
       <label for="validationServer04">State</label>
-      <input type="text" class="form-control is-invalid" id="validationServer04" placeholder="State" required>
+      <select class="form-control is-invalid" id="validationServer04" required>
+        <option selected disabled value = ''>Choose...</option>
```
Comment (mdo) [inline site/docs/4.3/components/forms.md]:
```suggestion
        <option selected disabled value="">Choose...</option>
```

File: site/docs/4.3/components/forms.md [hunk 12]
```diff
@@ -1075,21 +1084,24 @@ If your form layout allows it, you can swap the `.{valid|invalid}-feedback` clas
...
+      <input type="text" class="form-control" id="validationTooltip03" required>
       <div class="invalid-tooltip">
         Please provide a valid city.
       </div>
     </div>
     <div class="col-md-3 mb-3">
       <label for="validationTooltip04">State</label>
-      <input type="text" class="form-control" id="validationTooltip04" placeholder="State" required>
+      <select class="form-control" id="validationTooltip04" required>
+        <option selected disabled value = ''>Choose...</option>
```
Comment (mdo) [inline site/docs/4.3/components/forms.md]:
```suggestion
        <option selected disabled value="">Choose...</option>
```

**Review by mdo — COMMENTED** [review]
Please be sure to follow code standards used everywhere in this project—no spaces around attribute's `=` and double quotes around attribute values. Fixed these here with suggested changes.

**Comment by tojimjiang** [comment]
I have updated the validated selects to use `.custom-select`, and attribute values to follow the code standards.

**Review by patrickhlauke — APPROVED** [review]

**Comment by XhmikosR** [comment]
@patrickhlauke: should we backport this to v4?

**Comment by patrickhlauke** [comment]
> should we backport this to v4?

yes

**Comment by XhmikosR** [comment]
OK, I'll cherry pick this to the new v4-dev branch.

**Pull request merged**